### ISSUE TITLE
wrap* is not public

### ISSUE THREAD

**Issue by luxbock** [roomkey/annotate#1]
I'm getting the following error when trying to use the wrap macros with my code: 

`CompilerException java.lang.IllegalStateException: var: #'annotate.wrap/wrap* is not public`

**Comment by pbostrom** [comment]
Can you post a code snippet? This works for me:
```clojure
foo> (defn foo [x] (inc x))
#'foo/foo
foo> (foo :foo)
ClassCastException clojure.lang.Keyword cannot be cast to java.lang.Number  clojure.lang.Numbers.inc (Numbers.java:110)
foo> (annotate.wrap/wrapv foo [Int => Int])
#'foo/foo
foo> (foo :foo)
ExceptionInfo Failed to type check foo input(s): (not (integer? :foo))  clojure.core/ex-info (core.clj:4403)
```

**Comment by luxbock** [comment]
Whops, I should have been a bit more careful with my words.

The problem happens if I turn on the annotations in my `project.clj` with:
```clojure
:jvm-opts ["-Dannotate.typecheck=on"]
```
And then attempt to use the `wrap$` macro. `wrapv` appears to work just fine.

**Comment by pbostrom** [comment]
Ok, I see that now, thanks. I'll try to get this fixed.

**Comment by pbostrom** [comment]
Fixed in cfb61e36686f8949d7077d609008b6c05b4e058c
